torch/__init__.py

```diff
@@ -982,4 +982,4 @@ def _register_device_module(device_type, module):
 
     csan.enable_cuda_sanitizer()
 
-from ._dispatch import python
+torch._C._set_python_dispatcher(True)
```

Comment:
woah this got real fast

Reply:
this doesn't actually enable the dispatcher lol